Functions/xASL_bids_CompareStructures.m

```diff
@@ -47,14 +49,24 @@
     end
 	
     % Default value for bPrintReport
-	if nargin < 3 || isempty(bPrintReport)
+	if nargin < 3 || ~exist('bPrintReport','var')
```

Comment:
I really like the isempty solution. Henk recently changed a lot of my isempty input checks to exist, because he probably prefers that. Maybe we need to agree on something in the team meeting :)

Reply:
Wrote that on whatsapp. Isempty is really essential in the parameter admin. And exist is really useless - cause nargin does that already :) Yes - we need to discuss this and agree on a single solution. Especially because we frequently use [] - and this crashes the exist - or rather crashes after exist doesn't recognize the problem :)

Recently? Anything already merged, or we can still revert?